src/source-napatech.c

```diff
@@ -740,23 +742,23 @@ static void RecommendNUMAConfig(SCLogLevel log_level)
     }
 
     if (set_cpu_affinity) {
-        SCLog(log_level, __FILE__, __FUNCTION__, __LINE__,
+        SCLog(log_level, __FILE__, __FUNCTION__, __LINE__, _sc_module,
```

Comment:
In a follow up PR I would like to see this use of `SCLog` changed. Generally code is expected to use `SCLogDebug`, `SCLogNotice`, etc. But this can be done in a separate effort.

Reply:
I'll create a new PR for this

Reply:
I am working on a new PR for this - I've refactored this function so that it handles arbitrary number of NUMA nodes (the currently module is hardcoded up to 4 numa nodes).  Given that the AMD Epyc 9004 dual-socket servers can go up to 24 Numa nodes, it seems prudent to generalise.  I'll post it and rebase once we have finished this one.